pkg/api/types.go

```diff
@@ -614,6 +620,26 @@ type NFSVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// Represents a Quobyte mount that lasts the lifetime of a pod.
+// Quobyte volumes do not support ownership management or SELinux relabeling.
+type QuobyteVolumeSource struct {
+	// Registry represents a single or multiple Quobyte Registry services
+	// specified as a string as host:port pair (multiple entries are sperated with commas)
+	// which acts as the central registry for volumes
+	Registry string `json:"registry"`
+
+	// Volume is a string that references an already created Quobyte volume by name.
+	Volume string `json:"volume"`
+
+	// Path specifies the directory inside the Quobyte Volume that get's mounted
+	// if no path is provided the whole Quobyte volume is mounted
+	Path string `json:"path,omitempty"`
```

Comment:
You mean this https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1220-L1229 ? Is this a hidden feature or something else? I couldn't find any documentation about it. It looks like they are doing nearly the same thing just with one exception: If I specify a `VolumeMount.SubPath`  Kubernetes will still mount the whole Quobyte Volume on the host in contrast when specify `path` only the specified part of the Quobyte Volume gets mounted on the host.

Since it seems that `VolumeMount.SubPath`is a hidden feature I would like to have the `path`option in Quobyte. 

@quofelix any thoughts on this?
@thockin if the `VolumeMount.SubPath` is a feature for everyone we should document it (I could find't it in the [docs](http://kubernetes.io/docs/user-guide/volumes/) and in the [swagger-spec](http://kubernetes.io/kubernetes/third_party/swagger-ui/#!/api%2Fv1/createNamespacedPod).